### ISSUE TITLE
wget fails to scrape LW

### ISSUE THREAD

**Issue by fgloblek** [AABoyles/LessWrong-Portable#22]
Hi,

I encountered a problem with wget not working correctly. This occurred only on LW itself, with the blogs hosted elsewhere being unaffected. A brief check confirmed that running wget on any LW page exits with a 403 unless the user agent is specified. I guess they changed how their headers work at some point. 
I solved the issue locally by replacing line 81 of build.js with 
` execSync('wget --user-agent="Mozilla" ' + config.metadata.source + path + ' -nc -q -O ' + cache_path);  `
adding the `--user-agent="Mozilla" `. After this, wget doesn't get a 403 and everything runs correctly. 

For completeness, this was the result of running the script. Now it works fine. 

```
$ npm start epistemology

> lesswrong-portable@0.2.0-a start /home/fgloblek/Documents/LessWrong/LessWrong-Portable
> node build.js "epistemology"

Scraping https://www.lesswrong.com/s/SqFbMbtxGybdS2gRs/p/XqvnWFtRD2keJdwjX
child_process.js:669
    throw err;
    ^

Error: Command failed: wget https://www.lesswrong.com/s/SqFbMbtxGybdS2gRs/p/XqvnWFtRD2keJdwjX -nc -q -O ./cache/XqvnWFtRD2keJdwjX.html
    at checkExecSyncError (child_process.js:629:11)
    at execSync (child_process.js:666:13)
    at config.urls.forEach.url (/home/fgloblek/Documents/LessWrong/LessWrong-Portable/build.js:81:5)
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/home/fgloblek/Documents/LessWrong/LessWrong-Portable/build.js:71:13)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! lesswrong-portable@0.2.0-a start: `node build.js "epistemology"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the lesswrong-portable@0.2.0-a start script.
```

**Comment by AABoyles** [comment]
Awesome, thanks for working out the fix! Will put it in.

**Comment by AABoyles** [comment]
Resolved by 4894050a3b03b4ed8c865caf1b4dfe4499713be1
